README.adoc

```diff
@@ -70,7 +69,7 @@ $ brew install mvndaemon/homebrew-mvnd/mvnd
 
 We're looking for contribution to support https://www.macports.org[MacPorts],
 https://community.chocolatey.org/packages/mvndaemon/[Chocolatey], https://scoop.sh/[Scoop] or
-https://github.com/joschi/asdf-mvnd#install[asdf].  If you're fancy helping us...
+https://github.com/joschi/asdf-mvnd#install[asdf].  If you fancy helping us...
```

Comment:
-1 on this change. The original was correct IMO.

Reply:
I'm not a native speaker, but IIUC the original is incorrect, and the suggestion _is_ correct. See also examples e.g. [here](https://dictionary.cambridge.org/dictionary/english/fancy).

Reply:
I can see it being written as "If you are fancy, help us" or something along those lines. But I figured that the correction was your intended turn of phrase.

[Another example](https://hinative.com/questions/16176373) here the answers seem to change the you're to you

Reply:
Thanks for pointing to an evidence, gentlemen. I did not know "fancy" was also a verb.